flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/generated.scala

```diff
@@ -35,7 +35,11 @@ case class GeneratedExpression(
     resultTerm: String,
     nullTerm: String,
     code: String,
-    resultType: TypeInformation[_])
+    resultType: TypeInformation[_]) {
+
+  var leftNullTerm: String = _
```

Comment:
@fhueske It doesn't work because `${codeGenerator.input1Term}` is Row
`        @Override
        public void join(Object _in1, Object _in2, org.apache.flink.util.Collector c) throws Exception {
          org.apache.flink.types.Row in1 = (org.apache.flink.types.Row) _in1;
          org.apache.flink.types.Row in2 = (org.apache.flink.types.Row) _in2;`
But I need know which field in join expression is null